TestProject/output/ConsoleOutput.php

```diff
@@ -20,10 +20,15 @@ class ConsoleOutput extends BaseOutput
      */
     function outputBoard(Board $_board)
     {
+        $board = $_board->getBoard();
+
         for ($y = 0; $y < $_board->getHeight(); ++$y) {
             for ($x = 0; $x < $_board->getWidth(); ++$x) {
-                $_board->printOutBoard();
+                $currentBoard = $board[$x][$y] ? " * " : " - ";
```

Comment:
Just a Hint: You could also implement a getCell() method, this would make the code a little bit safer and cleaner.